src/plugins/docflow-smartfield/docflow-smartfield-ui.js

```diff
@@ -39,7 +39,7 @@ export default class DocflowSmartfieldUI extends Plugin {
 				tooltip: true
 			} );
 
-			this.listenTo( button, 'execute', () => this.insertNewSmartfield( TYPE.String ) );
```

Comment:
why did you remove the type from here?

Reply:
I intend to set type only on the oerwrite app. I set the default value to `SmartField::String` in the `NewTemplateSmartfield` component, so the behaviour is the same. This helps to control when to create a new smart field with backend API
The `useEffect` hook in `NewTemplateSmartfield` component checks if `type` is undefined before creating a new smartfield with the backend api. If type is set here then a new smartfield will not be created when the toolbar smartfield button is clicked.
https://github.com/dcflw/oerwrite/pull/9634/files